crates/rspack_plugin_html/src/config.rs

```diff
@@ -107,7 +116,7 @@ impl Default for HtmlPluginConfig {
       template: None,
       template_content: None,
       template_parameters: None,
-      inject: None,
+      inject: default_inject(),
```

Comment:
Why not keep `None`?

Reply:
I read the source code of [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin),  the default value of inject is 'true'.  I'll change it.  
Doing this, you can reduce `create_script` and `create_style` function。

the typescript side handles the case of `undefined`  in  [packages/rspack/src/config/builtins.ts](https://github.com/web-infra-dev/rspack/pull/3771/files#diff-908957625f9952a51b579835fc42cf2a8d717ac0ffa1ce16aa4ac24bf6362885)